website/api/trusted-setup-ceremony.md

```diff
@@ -0,0 +1,70 @@
+
+If you are looking for your own contribution, you can also go to gist.github.com and navigate to your Gist named "`risc-zero-stark-to-snark-prover_attestation.log`" (which will be linked at the top if you don't make Gists for other reasons; otherwise you can look for it under "View your gists" or with the search function as described in the previous paragraph). You can also find your contribution the same way as for any other user (i.e. by searching the transcript for your username).
+
+Important Note: Contributors can remove their attestations from GitHub at any time. They can also edit their attestations (although in this case the edit history will be visible). _Only the original version of the attestation can be valid; an edited version cannot be a valid attestation_. Note that that if any malicious contributors were able to participate in the ceremony, it does not damage the security of the ceremony, but it _does_ mean that they can pretend to have a bad attestation by editing or deleting their Gist. Therefore, a contribution with no attestation provides no security to the ceremony, but does not necessarily mean anything is wrong, either.
+
+Please exercise good judgment about whether a missing or edited attestation represents:
+1. A malicious contributor
+1. Someone just cleaning up old Gists
+1. A problem in the ceremony
+
+## Ceremony Security
+
+We used the open-source tools [p0tion] and [DefinitelySetup] to run our ceremony, and our ceremony was coordinated by the [PSE] team. This gave us tools that had been battle-tested by prior ceremonies, and moreover, by using tools written by an external team, we put substantial limits on our own ability to maliciously manipulate the ceremony software.
+
+
+
+[DefinitelySetup]: https://github.com/privacy-scaling-explorations/DefinitelySetup
+[install-circom]: https://docs.circom.io/getting-started/installation/
+[kobi-bad-ceremony-list]: https://twitter.com/kobigurk/status/1782502969453494530
+[p0tion]: https://github.com/privacy-scaling-explorations/p0tion
+[p0tion-config]: https://github.com/risc0/risc0/blob/d4e427283027c28b38b8eda1562e8e0e68d1b0e2/compact_proof/groth16/p0tionConfig.json
+[powers-of-tau-hez-23]: https://storage.googleapis.com/zkevm/ptau/powersOfTau28_hez_final_23.ptau
+[powers-of-tau-hez-23-our-mirror]: https://risc0-artifacts.s3.us-west-2.amazonaws.com/tsc/2024-04-04/powersOfTau28_hez_final_23.ptau
+[PSE]: https://pse.dev
+[pse-risc0-ceremony]: https://ceremony.pse.dev/projects/RISC%20Zero%20STARK-to-SNARK%20Prover
+[r1cs-file]: https://risc0-artifacts.s3.us-west-2.amazonaws.com/tsc/2024-04-04/stark_verify.r1cs
+[risc0-circom-library]: https://github.com/risc0/risc0/blob/d4e427283027c28b38b8eda1562e8e0e68d1b0e2/compact_proof/groth16/risc0.circom
+[snarkjs]: https://github.com/iden3/snarkjs
+[stark-verify-circom]: https://github.com/risc0/risc0/blob/d4e427283027c28b38b8eda1562e8e0e68d1b0e2/compact_proof/groth16/stark_verify.circom
```

Comment:
hmm... raising the question of whether this is the right place to point for this. 

do we have a more enshrined source of truth than `blob/d4e427283027c28b38b8eda1562e8e0e68d1b0e2`? 

if we have put this circom file into the amazon bucket, we might point there instead.

Reply:
No, in that this is from after the v0.21 release and so is not enshrined any release. It's also not on AWS (since it's checked in to GitHub). I originally pointed at `main`, but that isn't guaranteed to be stable; I can forward you a thread discussing this more.